packages/logger/README.md

```diff
@@ -489,23 +489,43 @@ Different types of data are serialized differently before being output as JSON:
       // }
       ```
 
-      Errors found in sub-properties of the log data are _not_ serialized like this. This is for performance reasons: looping over every nested property to check if it's an error is expensive. Do **not** do this:
+    Also note that errors found in sub-properties or as top-level errors are now automatically serialized as well:
```

Comment:
**suggestion:** reading only this line it suggests that we now serialize _all_ top-level properties which isn't true. I'd be tempted to leave this as-is but note the one exception. Maybe something like:

```suggestion
    Errors found in sub-properties of the log data are _not_ serialized like this except when the property name is `error`. This is for performance reasons: looping over every nested property to check if it's an error is expensive. Do **not** do this:
```

We'd then want to change the example below back, but because we have inconsistencies with `err` we probably want to change to something we definitely don't serialize, e.g.

> ```js
>  logger.info({ myErrorProperty: new Error('Oops') });
>  // Outputs:
>  // {
>  //     "level": "info",
>  //     "myErrorProperty": {}
>  // }

Same goes for the little bit after this, let's make it super clear by removing the `err` property from the examples and change it to something that we'll never serialize (e.g. `myErrorProperty`)

Reply:
Good point, will change to clarify the sub-property serialization